doc/scripts/requirements.txt

```diff
@@ -1,5 +1,5 @@
 breathe==4.9.1
-sphinx==1.7.5
+sphinx==1.7.7
 docutils==0.14
 sphinx_rtd_theme
```

Comment:
Would it make sense to also add a version to this, i.e. `sphinx_rtd_theme==0.4.0`